docs/testing/9_performance.md

```diff
@@ -0,0 +1,76 @@
+# Performance
+
+If the battery of tests in a project starts taking too long to complete, here's a few tips to improve performance:
+
+# setUpTestData and setUpClass
+
+If you have several test_ methods in your TestCase class and all of them (or most of them) are using data created in the method "setUp", you should pass that data creation to the method setUpTestData. Here's some key points about setUpTestData:
+
+- It's not a method of unittest, but it's a django creation that was created to help speed up test execution.
```

Comment:
nit: django abstraction?

Reply:
It's not an abstraction, it's just a classmethod that was created by Django and inserted into the lifecycle of the tests.